src/CacheKey.php

```diff
@@ -174,12 +174,16 @@ protected function getValuesFromWhere(array $where) : string
 
     protected function getValuesFromBindings(array $where, string $values) : string
     {
-        if (($this->query->bindings["where"][$this->currentBinding] ?? false) !== false) {
-            $values = $this->query->bindings["where"][$this->currentBinding];
+        // Fallback to this when the current binding does not exist in the bindings array
+        $bindingFallback = __CLASS__ . ':UNKNOWN_BINDING';
+        $currentBinding = $this->getCurrentBinding('where') ?? $bindingFallback;
```

Comment:
Thanks for the update! What I wanted to avoid was using `?? $bindingFallback`. Let me see if I can edit it.